pkg/rootless/rootless_linux.go

```diff
@@ -242,9 +243,22 @@ func BecomeRootInUserNS() (bool, int, error) {
 		}
 		if err == nil {
 			username = user.Username
+			groupname = username
+			group, err := gouser.LookupGroupId(user.Gid)
+			if err == nil {
+				groupname = group.Name
+			}
+		}
+	} else {
+		user, err := gouser.Lookup(username)
+		if err == nil {
+			group, err := gouser.LookupGroupId(user.Gid)
+			if err == nil {
+				groupname = group.Name
+			}
```

Comment:
If you fail here in LookupGroupId, would it be better to set groupname to user.Username (which just succeeded) rather than the username you got from `OS.GetEnv()` at line 234?

Reply:
But it's all just guessing ? Just defaulted to the old value, in case sanity fails

Reply:
Would t `groupname=fmt.Sprintf("%d", user.GID)` work?